vincentlanglet/twig-cs-fixer/3.0/post-install.txt

```diff
@@ -0,0 +1,4 @@
+<bg=green;fg=white></>
+<bg=green;fg=white> You can edit the `.twig-cs-fixer.dist.php` to your needs </>
```

Comment:
You can create a `.twig-cs-fixer.dist.php` or something like this ?

Reply:
Update this line :)